stylepak

```diff
@@ -56,7 +56,7 @@ done
 [[ -n "$theme" ]] || \
   theme=$(gsettings get org.gnome.desktop.interface gtk-theme | tr -d "'")
 
-echo "$theme" | grep -Eq '^[A-Za-z0-9_\-]+$' || \
+echo "$theme" | grep -Eq '^[A-Za-z0-9._\-]+$' || \
   die "Invalid theme name (may only contain letters, digits, '_', '-'): $theme"
```

Comment:
Could the error here be updated to mention dots?

Reply:
Yes, why not.